### PR TITLE
update ja/docs/concepts/policy/resource-quotas.md

### LINKED ISSUE #26878: ja: Make docs/concepts/policy/resource-quotas.md follow v1.19 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/concepts/policy/resource-quotas.md to follow v1.19 of the original (English) text.

**Why is this needed**
content/ja/docs/concepts/policy/resource-quotas.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.1/content/ja/docs/concepts/policy/resource-quotas.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/concepts/policy/resource-quotas.md

diff between translated and v1.19
https://gist.github.com/5c61b115c1eb0f8adda192435bb1b2a3

**Note**
Currently, we use **dev-1.19-ja.1** branch. Please open a PR targeting the branch.

ref
https://kubernetes.io/ja/docs/contribute/localization/

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by s-kawamura-w664**
Updated ja/docs/concepts/policy/resource-quotas.md to follow v1.19 of the original (English) text.
Fixes #26878

### CODE REVIEW

**Comment by s-kawamura-w664** [comment]
/assign @inductor
Could you please review?
If this is OK, please approve.

File: content/ja/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -148,27 +152,73 @@ Kubernetes v1.9より前のバージョンでは、限定されたリソース
...
-* `limits.memory`
 * `memory`
-* `pods`
 * `requests.cpu`
 * `requests.memory`
+* `limits.cpu`
+* `limits.memory`
+
+同じクォータで`Terminating`と`NotTerminating`の両方のスコープを指定することはできず、また
+同じクォータで`BestEffort`と`NotBestEffort`の両方のスコープを指定することもできないことに注意してください。
```
Comment (inductor) [inline content/ja/docs/concepts/policy/resource-quotas.md:172]: ここ改行けしてもらえると。Markdownで半角スペースが入っちゃうと思うんで

**Review by inductor — CHANGES_REQUESTED** [review]
一点だけ。

**Comment by s-kawamura-w664** [comment]
@inductor 
I updated according to your review comment.
Could you please check it?

**Pull request merged**